src/documents/posts/2015-04-02-delivery-engineering.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Delivery Engineering
+author: Bob Zoller
+layout: post
+disqus:
+  shortname: goodeggsbytes
+  url: "http://bites.goodeggs.com/post/delivery-engineering"
+---
+
+Delivery Engineering is a term I first heard in the January 2015 [Thoughtworks Radar][thoughtworks-deleng]:
+
+> where there *is* a need for significant investment in [DevOps] tooling and automation, we do see a role for a Delivery Engineering team.  Rather than being a helpdesk, these teams build tooling and enable teams to deploy, monitor, and maintain their own production environments.
+
+In other words, Delivery Engineering enables the _culture_ of DevOps to thrive on each team, rather than silo that knowledge up in one team or role.
+
+I was excited when I read this because it concisely captured the goals of a team I had recently formed, which we were calling (quite opaquely), "Infrastructure."  We formed the Infrastructure team because we felt like we had crossed a threshold as an engineering organization, where it made sense to carve out a set of folks that could own shared tools and services used by all our engineering teams.  Think continuous integration pipelines, node modules for reporting stats, best practices for alerting.
```

Comment:
> Think continuous integration pipelines, node modules for reporting stats, best practices for alerting.

grammer: not  a complete sentence, maybe you want to attach it to the previous with an m-dash?